keys/storage/filestorage/main.go

```diff
@@ -124,47 +128,81 @@ func (s FileStore) nameToPaths(name string) (pub, priv string) {
 	return path.Join(s.keyDir, pubName), path.Join(s.keyDir, privName)
 }
 
-func writeInfo(path string, info keys.Info) error {
-	return write(path, info.Name, info.PubKey.Bytes())
-}
-
 func readInfo(path string) (info keys.Info, err error) {
```

Comment:
I think we really need to change this so the info is easier to parse without our code, but that is another issue.
For the status quo - readInfo/writeInfo are fine